fleet/lib/cnpg-cluster/overlays/pillan/cluster-cnpg-cluster.yaml

```diff
@@ -0,0 +1,43 @@
+---
+# Cluster Definition
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  name: cnpg-cluster
+spec:
+  imageName: docker.io/lsstit/cnpgsphere:14.5
+  imagePullPolicy: IfNotPresent
```

Comment:
I'm assuming this is intentionally different from the value on ruka?

Reply:
yes, in ruka is set to always pull for testing purposes 👍🏻